client/src/components/CreateCard/cardContext.js

```diff
@@ -148,10 +164,31 @@ const CardProvider = props => {
     }
   };
 
+  const handleOpenDelete = () => {
+    setOpenDelete(true);
+  };
+  const handleCloseDelete = () => {
+    setOpenDelete(false);
+  };
+
+  const handleDelete = history => {
+    authFetch("", {
```

Comment:
Probably its intentional, for just in case I'll report this.
delete route seems not to point correct route.
I wanna make sure if we can actually delete card with FE in this branch.
Because I could not find the button for it. 
The pick attached is the app with this pr
<img width="848" alt="Screen Shot 2020-03-01 at 8 04 18 PM" src="https://user-images.githubusercontent.com/38660310/75638265-03fb3300-5bf9-11ea-9671-bbe51d33500b.png">

Reply:
delete button will only show up once you make a card

Reply:
HI, I could make sure delete button works. 
Pleas merge this to dev.